README.md

```diff
@@ -46,13 +46,12 @@ Weaviate makes it easy to use state-of-the-art ML models while giving you the sc
    <br><sub></sub>
 
 * **Any media type with Weaviate Modules**<br>
-  Use State-of-the-Art ML model inference (e.g. Transformers) for Text, Images, etc. at search and query time to let Weaviate manage the process of vectorizing your data for your - or import your own vectors.
+  Use State-of-the-Art ML model inference (e.g. Transformers) for Text, Images, etc. at search and query time to let Weaviate manage the process of vectorizing your data for your - or import your vectors.
```

Comment:
Hi @omarzain27, I agree with all the changes except this one. I think here we lose meaning if we don't explicitly express "your own" for the vectors to differentiate it from auto-generated vectors. However, there is another grammatical error in this snippet which is "data for you" instead of "data for your". So I think this snippet should be:

> Use State-of-the-Art ML model inference (e.g. Transformers) for Text, Images, etc. at search and query time to let Weaviate manage the process of vectorizing your data for you - or import your own vectors.